php/libraries/Project.class.inc

```diff
@@ -75,7 +75,7 @@ class Project
 
                 $project = new Project();
 
-                $project->_projectId         = $projectData['projectId'];
+                $project->_projectId         = (int) $projectData['projectId'];
```

Comment:
Can you add a `@var int` annotation at the definition of `$project->_projectId` so that phan can detect if there are any other usages where the the type conflicts?

Reply:
@driusan thanks and added.